### PR TITLE
Add Table for Laporan Attachments

### PR DESCRIPTION
**Pull request by allisandrana**
Add Missing Table Attachment, this will store attachment file in blob (bytea) format.

### CODE REVIEW

**Review by izuleon — DISMISSED** [review]
LGTM

**Comment by Ujang360** [comment]
Please add @izuleon and @allisandrana in the [contributor(s) section of the README.md](https://github.com/QTILabs/numbering-engine/blob/develop/README.md#contributors)

File: src/schema.rs [hunk 1]
```diff
@@ -1,3 +1,12 @@
+table! {
+    attachment (id) {
+        id -> Int8,
```
Comment (Ujang360) [inline src/schema.rs]:
```suggestion
        id -> Int4,
```

**Review by Ujang360 — CHANGES_REQUESTED** [review]

**Review by Ujang360 — APPROVED** [review]
LGTM

**Review by yosiasm — APPROVED** [review]
lgtm

**Pull request merged**